shared/actions/wallets.js

```diff
@@ -778,6 +784,7 @@ function* walletsSaga(): Saga.SagaGenerator<any, any> {
   yield Saga.actionToPromise(WalletsGen.loadDisplayCurrencies, loadDisplayCurrencies)
   yield Saga.actionToPromise(WalletsGen.loadSendAssetChoices, loadSendAssetChoices)
   yield Saga.actionToPromise(WalletsGen.loadDisplayCurrency, loadDisplayCurrency)
+  yield Saga.actionToAction(WalletsGen.loadDisplayCurrency, refreshAssets)
```

Comment:
we want 
DisplayCurrenciesReceived and not loadDisplayCurrency

Reply:
ok thanks